fcl/prolog.fcl

```diff
@@ -173,7 +173,7 @@ TrkAnaTreeMaker : {
   FillTriggerInfo : true
   TriggerProcessName : "Mix"
   ProcessEmptyEvents : false
-  FillCRVHits : true
+  FillCRVCoincs : true
   FillCRVPulses : false
```

Comment:
Should we also change ```FillCRVPulses``` to ```FillCRVDigis``` too?

Reply:
In the current scheme I think this comes down to preference. `FillCRVPulses` switches on both the `crvdigis` and `crvpulses` branches, since these are both filled in `CrvInfoHelper::FillCrvPulseInfoCollections`:

```
      if(_fillcrvpulses){
        _crvHelper.FillCrvPulseInfoCollections(_crvRecoPulses, _crvDigiMCs, _crvDigis,
                                               _crvpulses, _crvpulsesmc, _crvdigis);
      }
```

I'm not sure why we do it this way to be quite honest, you could split `digis` and `pulses` up and have two different flags. That might make things less confusing?

Reply:
Sounds good. If it's turns out to be too much hassle to separate it out, then I think I would prefer ```digis``` over ```pulses``` in the fcl parameter name. We're also thinking about updating the parameter names for the flags that the tracker uses for its digis so some consistency there would be good

Reply:
Should I just push what I've done on this comment even if I'm not sure if it's an improvement? Is it easy to remove commits from PRs? Thanks

Reply:
To "undo" a commit you create the inverse commit and commit it.  There is a command to create the inverse commit.  So it's always there in the history but so long as you end up in the right place in the end, that's what you want.